tests/functional/test_s3.py

```diff
@@ -28,6 +27,7 @@
     UnsupportedS3AccesspointConfigurationError,
     UnsupportedS3ConfigurationError,
 )
+from botocore.httpchecksum import HAS_CRT, Crc32Checksum, CrtCrc32Checksum
```

Comment:
We shouldn't be doing transitive imports. `HAS_CRT` comes from botocore.compat, so we should add that on line 21.
```suggestion
from botocore.httpchecksum import Crc32Checksum, CrtCrc32Checksum
```